Web Quiz Engine (Kotlin)/task/src/engine/WebQuizRepository.kt

```diff
@@ -8,10 +10,7 @@ import java.util.concurrent.ConcurrentMap
 typealias QuizIdentifier = Int
 
 @Component
-class WebQuizRepository {
-
-    @Autowired
-    private lateinit var idGenerator: UniqueIdGenerator
+class WebQuizRepository(private val idGenerator: UniqueIdGenerator) {
```

Comment:
Wow very nice utilization on dependency injection! I'm surprised and proud.

Reply:
Thks you. I feel very smart right now 😊